script/options.js

```diff
@@ -132,6 +140,7 @@ function applyBrandConsolidationOverrides(options) {
 function getOptions() {
   const options = gatherFromCommandLine();
 
+  applyDeprecatedBuildtypes(options);
```

Comment:
I think just adding another step here is the clearest way to apply the redirect behavior.